src/modules/notifications/entities/notifications.entity.ts

```diff
@@ -0,0 +1,15 @@
+import { AbstractBaseEntity } from '../../../entities/base.entity';
+import { User } from '../../user/entities/user.entity';
+import { Column, Entity, ManyToOne } from 'typeorm';
+
+@Entity()
+export class Notification extends AbstractBaseEntity {
+  @Column({ nullable: false })
+  message: string;
+
+  @Column({ nullable: false, default: false })
+  isRead: boolean;
```

Comment:
change to `is_read`. Also look for other instances where you use `isRead` and change them